src/com/shapeshifters/thecrash/controller/TheCrashApp.java

```diff
@@ -22,6 +22,7 @@
     private Map<String, Room> rooms;
     private Map<String, String> verbs;
     private Map<String, String> directions;
+    private Collection<String> items = new ArrayList<>();
```

Comment:
Where are you using this?

Reply:
I forgot to delete that, Ill get rid of it